config/galaxy.ini.sample

```diff
@@ -633,6 +638,29 @@ nglims_config_file = tool-data/nglims.yaml
 # https://FQDN/galaxy-prefix/gie_proxy for proxying
 #dynamic_proxy_prefix=gie_proxy
 
+# The Golang proxy also manages the docker containers more closely than the
+# NodeJS proxy, so is able to expose more container management related options
+
+# This attribute governs the minimum length of time between consecutive HTTP/WS
+# requests through the proxy, before the proxy considers a container as being
+# inactive and kills it.
+#dynamic_proxy_golang_noaccess = 60
+
+# In order to kill containers, the golang proxy has to check at some interval
+# for possibly dead containers. This is exposed as a configurable parameter,
+# but the default value is probably fine.
+#dynamic_proxy_golang_clean_interval = 10
+
+# The golang proxy needs to know how to talk to your docker daemon. Currently
```

Comment:
This PR tries to centralise all of that configuration into one place. It seems unpleasant for admins to maintain N config files with docker details. 

I was waiting for feedback from you mostly, since I don't know if anyone else runs different images on different machines. If that's really absolutely necessary, then I'll have to re-work a lot of things. 

I'm somewhat opinionated on this topic (unfortunately) but I see less value for admins in saying "this box will only run rstudio" or "this box will only run jupyter images", why aren't they using docker swarm or similar in distributing across their infrastructure.